Removal of empty username support in keyring 25.3.0 breaks several tests

```
$ gh repo clone jaraco/keyrings.alt
$ cd keyrings.alt
$ tox
[…]
=========================================================================================== short test summary info ============================================================================================
FAILED tests/mocks.py::ruff
FAILED tests/test_crypto.py::TestCryptedFileKeyring::test_empty_username - ValueError: Username cannot be blank.
FAILED tests/test_file.py::TestEncryptedFileKeyring::test_empty_username[Crypto] - ValueError: Username cannot be blank.
FAILED tests/test_file.py::TestEncryptedFileKeyring::test_empty_username[Cryptodome] - ValueError: Username cannot be blank.
FAILED tests/test_file.py::TestUncryptedFileKeyring::test_empty_username - ValueError: Username cannot be blank.
================================================================================== 5 failed, 154 passed, 81 skipped in 9.05s ===================================================================================
```

--- a/tests/mocks.py
+++ b/tests/mocks.py
@@ -141,7 +141,7 @@ class MockDocumentService(MockGDataService):
                     delattr(self, '_put_err')
             else:
                 put_err = self._put_err
-            if type(put_err) == tuple:
+            if type(put_err) is tuple:
                 raise put_err[0](put_err[1])
             else:
                 raise put_err()
@@ -161,7 +161,7 @@ class MockDocumentService(MockGDataService):
 
     def request(self, data, uri):
         if hasattr(self, '_request_err'):
-            if type(self._request_err) == tuple:
+            if type(self._request_err) is tuple:
                 raise self._request_err[0](self._request_err[1])
             else:
                 raise self._request_err()

--- a/tests/test_file.py
+++ b/tests/test_file.py
@@ -151,6 +151,10 @@ class FileKeyringTests(BackendBasicTests):
         self.save_config(config)
         assert self.keyring._check_version(config) is False
 
+    def test_empty_username(self):
+        with pytest.raises(ValueError):
+            self.set_password('service1', '', 'password1')
+
 
 @pytest.fixture(scope="class")
 def monkeyclass(request):

